modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportObjectDefinitionTest.java

```diff
@@ -166,6 +150,58 @@ private MockLiferayResourceRequest _createMockLiferayResourceRequest(
 		return mockMultipartHttpServletRequest;
 	}
 
+	private void _exportImportObjectDefinition(
+			String fileName, String name, boolean system)
+		throws Exception {
+
+		PermissionChecker permissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		PermissionThreadLocal.setPermissionChecker(
+			PermissionCheckerFactoryUtil.create(TestPropsValues.getUser()));
+
+		ObjectDefinitionResource.Builder builder =
+			_objectDefinitionResourceFactory.create();
+
+		ObjectDefinitionResource objectDefinitionResource = builder.user(
+			TestPropsValues.getUser()
+		).build();
```

Comment:
@pedro-oliveira446 why do we need to build this ObjectDefinitionResource?, I mean, we already have the _objectDefinitionResource injected in this class, is there any specific reason?

Reply:
@MarinhoFeliphe I changed the code to use the variable you said but I don't know if I did it the way you thought, can you check it?